operator/v1/tests/networks.operator.openshift.io/AdditionalRoutingCapabilities.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "Network"
+crdName: networks.operator.openshift.io
+featureGate: AdditionalRoutingCapabilities
+tests:
+  onCreate:
+  - name: Should be able to create a minimal Network
+    initial: |
+      apiVersion: operator.openshift.io/v1
+      kind: Network
+      spec: {} # No spec is required for a Network
+    expected: |
+      apiVersion: operator.openshift.io/v1
+      kind: Network
+      spec:
+        disableNetworkDiagnostics: false
+        logLevel: Normal
+        operatorLogLevel: Normal
+  - name: Should enable FRR advanced routing provider
```

Comment:
I have a negative case here related to the only custom validation that we have in place
https://github.com/openshift/api/pull/1944/files#diff-2e5838ab85bfbd5fa416b0be7d9f0cb258adbfe7af7bce2605383505458a4743R45

However when I tried to add negative tests for other built-in validations, the test itself fails to execute just because of the invalid input so I can really verify those. Maybe you had in mind other tests that did not occur to me or I did something wrong?

Reply:
I would typically expect a validation failure showing that a non Enum value cannot be set, we have those for other APIs

Reply:
let me try that